installer/charts/rhtap-pipelines/values.yaml

```diff
@@ -1,2 +1,3 @@
 ---
-pipelines: {}
\ No newline at end of file
+pipelines:
+  namespace: __OVERWRITE_ME__
```

Comment:
Not understanding this value, why not setting to the default that was hardcoded in the template "openshift-pipelines"

Reply:
This is our convention for saying "this value comes from values.yaml.tpl and/or config.yaml".
We could choose to hardcode the value in the future if there's no benefit to having it as a variable.

Reply:
Understand now